### PR TITLE
Improve define own property.

### PR DESCRIPTION
**Pull request by zherczeg**

### CODE REVIEW

File: jerry-core/ecma/operations/ecma-objects-general.c [hunk 1]
```diff
@@ -476,80 +479,87 @@ ecma_op_general_object_define_own_property (ecma_object_t *obj_p, /**< the objec
...
-                                                              property_name_p,
-                                                              prop_attributes,
-                                                              NULL);
+      JERRY_ASSERT (current_property_type == ECMA_PROPERTY_TYPE_NAMEDACCESSOR);
+#ifdef JERRY_CPOINTER_32_BIT
+      ecma_getter_setter_pointers_t *getter_setter_pair_p;
+      getter_setter_pair_p = ECMA_GET_POINTER (ecma_getter_setter_pointers_t,
+                                               ECMA_PROPERTY_VALUE_PTR (property_p)->getter_setter_pair_cp);
+      jmem_pools_free (getter_setter_pair_p, sizeof (ecma_getter_setter_pointers_t));
+#endif
```
Comment (LaszloLango) [inline jerry-core/ecma/operations/ecma-objects-general.c]: missing comment: `#endif /* JERRY_CPOINTER_32_BIT */`

File: jerry-core/ecma/base/ecma-globals.h [hunk 2]
```diff
@@ -452,12 +452,22 @@ typedef union
 } ecma_property_ref_t;
 
 /**
+ * Extended property reference.
```
Comment (LaszloLango) [inline jerry-core/ecma/base/ecma-globals.h]: IMHO a bit more detailed description would be good for this.

**Comment by zherczeg** [comment]
Patch updated

**Comment by LaszloLango** [comment]
Thanks for the update. LGTM

**Pull request merged**